flow/connectors/mysql/qvalue_convert.go

```diff
@@ -290,6 +293,16 @@ func QValueFromMysqlFieldValue(qkind qvalue.QValueKind, fv mysql.FieldValue) (qv
 		v := fv.AsString()
 		unsafeString := shared.UnsafeFastReadOnlyBytesToString(v)
 		switch qkind {
+		case qvalue.QValueKindUInt64: // bit
```

Comment:
comment for why it's here?

Reply:
already commented, for qrep reference is https://github.com/go-mysql-org/go-mysql/blob/master/mysql/rowdata.go